packages/core/src/analytics/payment-providers-tracker/noop-payment-providers-tracker.ts

```diff
@@ -0,0 +1,19 @@
+import PaymentProvidersTrackerService from './payment-providers-tracker-service';
+
+export default class NoopPaymentProvidersTracker implements PaymentProvidersTrackerService {
+    checkoutBegin(): void {}
+    clickPayButton(): void {}
+    customerEmailEntry(): void {}
+    customerSuggestionInit(): void {}
+    customerSuggestionExecute(): void {}
+    customerPaymentMethodExecuted(): void {}
+    exitCheckout(): void {}
+    orderPurchased(): void {}
```

Comment:
Do we really need all these methods/events? or maybe we can focus on the `selectedPaymentMethod` only?

Reply:
This is a mock for PaymentProvidersTracker (a copy of bodl service UI part), so we can use this methods in the future. Also for BraintreeConnect tracker we will use only 2 methods: `selectPaymentOption`, `walletButtonClick` events

Reply:
So in our case we have a focus on what PayPal asked for

Reply:
I agree with @BC-ACherednichenko that we should start with the basic methods now and not copy BODL here. Maybe we can keep the interface to only two methods for now.